DiscordCardLinker/CardBot.cs

```diff
@@ -154,103 +184,98 @@ public class CardBot {
 				MinimumLogLevel = LogLevel.Debug
 			});
 			Client.MessageCreated += OnMessageCreated;
-			Client.MessageReactionAdded += OnReactionAdded;
+			Client.ComponentInteractionCreated += OnButtonPressed;
+			Client.ThreadCreated += OnThreadCreated;
+			Client.ThreadUpdated += OnThreadUpdated;
 
 			// https://dsharpplus.github.io/api/DSharpPlus.DiscordClient.html?q=ConnectAsync#DSharpPlus_DiscordClient_ConnectAsync_DiscordActivity_System_Nullable_UserStatus__System_Nullable_DateTimeOffset__
 			/* connecting to discord */
 			await Client.ConnectAsync();
 			/* successfully connected to discord */
 		}
 
-		/*
-		 * process reaction to discord chat message
```

Comment:
Please do not remove comments describing what the code does.

Reply:
That function was removed and so the orphan comment with it.  I can add comments to the currently uncommented functions.